src/base/bittorrent/session.cpp

```diff
@@ -4405,7 +4409,7 @@ void Session::handleStateUpdateAlert(const lt::state_update_alert *p)
             ++m_torrentStatusReport.nbErrored;
     }
 
-    emit torrentsUpdated();
+    emit torrentsUpdated(updatedTorrents);
```

Comment:
There can be 0 updatedTorrents so you should prevent signal in that case.

Reply:
That cannot be done, there are other slots connected to it and expects this signal to be emitted at regular interval.

Reply:
Ok.